development/build/transforms/README.md

```diff
@@ -0,0 +1,123 @@
+# Local Browserify Transforms
```

Comment:
This is some _incredible_ documentation by the way. Not just this but the inline docs too.

I was expecting this transform to be difficult to understand, but it was really straightforward with all of these explanations at each step.

Reply:
+1000